### ISSUE TITLE
Rig control on Windows

### ISSUE THREAD

**Issue by xylo04** [la5nta/pat#380]
I received a question from Jim KE0NRE about Pat and rig control on Windows. On Linux we use Hamlib to achieve rig control, but that's not available on Windows (or Mac IIRC). This would be nice to have in the name of better cross-platform support.

**Comment by harenber** [comment]
Hamlib is available on Mac OS. Easily installable through homebrew.

**Comment by martinhpedersen** [comment]
I think hamlib and rigctld is supported on Windows as well? 🤔 Their [latest release](https://github.com/Hamlib/Hamlib/releases/tag/4.5) includes Windows binaries. I guess it's not the best UX for a Windows user since most Windows apps are monolithic, but I think the cost of adding and maintaining built-in rig control is not worth the effort at this point.

Maybe something similar to Pat Menu could make it easier for Windows users to manage the various services needed to operate a Pat on Windows. I think that would require much less effort in the long run.

**Comment by xylo04** [comment]
Oh look at that, they do include Windows binaries. I recall having trouble actually building Pat against the libraries, but probably I was doing something wrong. This may be a non-issue; I'll do some testing when I get a chance.

**Comment by martinhpedersen** [comment]
Yes, I imagine linking against the C library might be challenging without doing some patching. But Windows users should be able to use rigctld over TCP just like on any other platform, and I think that's our best bet 😊
